@navikt/core/css/form/suggest-field.css

```diff
@@ -0,0 +1,121 @@
+.navds-suggest-field {
+  display: flex;
+  flex-direction: column;
+}
+
+.navds-suggest-field__wrapper {
+  position: relative;
+}
+
+.navds-suggest-field__input {
+  padding-right: var(--navds-spacing-12);
+  padding-left: var(--navds-spacing-11);
+}
+
+.navds-form-field--small .navds-suggest-field__input {
+  min-height: 32px;
+  padding: 0.15rem;
+  padding-right: var(--navds-spacing-8);
+  padding-left: var(--navds-spacing-8);
+}
+
+.navds-suggest-field__button-clear {
+  position: absolute;
+  right: 0;
+  border-radius: var(--navds-border-radius-medium);
+  color: var(--navds-semantic-color-text);
+  height: 48px;
+  width: 48px;
+  top: 50%;
+  transform: translateY(-50%);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  cursor: pointer;
+  background: none;
+  border: none;
+  font-size: 1.5rem;
```

Comment:
Jeg synes det er rart at denne ikke er lik som clear-button i Search.

Reply:
I grunn synes jeg det er rart at vi har behov for to søkefelt. Er vi sikre på at vi ikke klarer oss med ett?

Reply:
Kan høre med designer om vi kan bruke samme clear-button for begge

> Er vi sikre på at vi ikke klarer oss med ett?

Slik jeg har forstått komponenten er det ganske forskjellig usecase for SuggestField og Search. 

SuggestField: 
- Autocomplete der resultater kommer opp i en popover under søkefeltet
- Som filter der søket oppdaterer resultatene som vises på siden fortløpende (eks ikonsiden vår)
- Kan brukes inline i tabeller eller InternHeader

Search:
- Resultat trenger ikke komme opp når man skriver
- Trykk på søkeknapp åpner gjerne en egen side eller søkemodal ved klikk (eks nav.no søket i dekoratør)
- Selve søket kan ta lengre tid da mye data må håndteres for finne resultat (eks data.nav.no)

Reply:
Det gir mening! Burde neste steg for denne komponenten da være å lage støtte for søkeforslag?

Reply:
Selve funksjonaliteten med popover/navigering pil opp/ned har vært kult å tilby med komponenten. Men det blir kanskje en egen komponent da som kan gjenbrukes for select/multiselect/vanlig søk etc

Reply:
Bruker nå samme clearbutton-styling som `Search` 🚀  Var bare ikke oppdatert i design.